api/migrations/0001_initial.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 4.2.4 on 2023-08-27 14:00
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='countryID',
+            fields=[
+                ('country_id', models.CharField(max_length=3, primary_key=True, serialize=False)),
+                ('country_name', models.CharField(max_length=100)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='pincode',
+            fields=[
+                ('pin_code', models.CharField(max_length=15, primary_key=True, serialize=False, unique=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='stateID',
+            fields=[
+                ('state_id', models.BigAutoField(primary_key=True, serialize=False)),
+                ('state_name', models.CharField(max_length=100)),
+                ('country_id', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='api.countryid')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='cityID',
+            fields=[
+                ('city_id', models.BigAutoField(primary_key=True, serialize=False)),
+                ('city_name', models.CharField(max_length=100)),
+                ('state_id', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='api.stateid')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='address',
+            fields=[
+                ('id', models.BigAutoField(primary_key=True, serialize=False)),
+                ('line1', models.TextField()),
+                ('line2', models.TextField()),
+                ('phone_number', models.CharField(max_length=15)),
```

Comment:
what if someone enters `abcd` as phone number?

Reply:
Yes it will work, CharField allows numeric, alphabets and special characters.

Some of the solutions for this is to use 'phonenumber_field' library or have a validator which validates that the input is a valid phone number. This validation can be done by creating a regex.